assets/src/edit-story/style.css

```diff
@@ -39,6 +39,7 @@ body.js.edit-story #web-stories-editor {
   background-color: #1b1d1c; /* theme.colors.bg.workspace */
   position: relative;
   height: calc(100vh - 32px); /* ADMIN_TOOLBAR_HEIGHT = 32 */
+  overflow: hidden;
```

Comment:
This one felt a bit dirty, but there was no other way around it. If you zoomed in to 300%, added a large image and then tried to drag in another image from library and tried snapping to the large image on screen, the snap guides would exceed the workspace and cause the entire browser window to have scrollbars. I couldn't find a good location in the editor to set this overflow, so this seemed to be the best location. It'll also fix any other weird overflow issues we might get in the future.

But I'm curious if anyone can see a problem with this?

Reply:
It shouldn't cause a problem at this point, the only thing it could do is hide some potential bugs with overflow that would require fixing in another way. There has been at least one overflow case where unused Moveable elements were left outside of the editor view and this caused overflow. Hiding overflow would have "fixed" it but it actually needed cleaning up the unused Moveables instead.

Reply:
I tried to break it, but couldn't. So far so good I suppose.